src/sentry/static/sentry/app/components/events/interfaces/frame/line.tsx

```diff
@@ -471,4 +416,47 @@ const LeadHint = styled('div')`
   width: 67px;
 `;
 
-export default withSentryAppComponents(Line, {componentType: 'stacktrace-link'});
+const StyledIconChevron = styled(IconChevron, {
+  shouldForwardProp: prop => prop !== 'isExpanded',
+})<{isExpanded: boolean}>`
+  transform: rotate(${p => (p.isExpanded ? '180deg' : '0deg')});
+  transition: 0.1s all;
+`;
+
+const ToogleContextButtonWrapper = styled('span')`
+  margin-left: ${space(1)};
+`;
+
+// the Buton's label has the padding of 3px because the button size has to be 16x16 px.
+const ToogleContextButton = styled(Button)`
+  span:first-child {
+    padding: 3px;
+  }
+`;
+
+const StyledListItem = styled(ListItem)`
+  padding-left: 0;
+  flex-direction: column;
+  align-items: flex-start;
+  ${PackageStatusIcon} {
+    flex-shrink: 0;
+  }
+  :hover {
+    ${PackageStatusIcon} {
+      visibility: visible;
+    }
+    ${AddressToggleIcon} {
+      visibility: visible;
+    }
+    ${FunctionNameToggleIcon} {
```

Comment:
```suggestion
    ${PackageStatusIcon},
    ${AddressToggleIcon},
    ${FunctionNameToggleIcon} {
```

I think you can combine all these selectors to share a property list as they are the same.